### DISCUSSION TITLE
MultiSelect does not accept object array as default only string array

### DISCUSSION THREAD

**Discussion by MariuzM** [mantinedev/mantine#7048] [Q&A]
On form load, i want to pass array object as default value to be preselected
```
[
	{ "label": "test", "value": "1" },
	{ "label": "test2", "value": "2" }
]
```

Only way todo is to pass array of strings `values` not `label` but with that it only shows ids but i want `label`

My MultiSelect works when its clicked then it loads from API all the options and then MultiSelect maps does ids with all options and shows the label correctly, but the problem is initial load of form shows ID's and not labels

**Comment by rtivital** [comment]
I do not understand the issue, MultiSelect value is array of strings that reference `value` of the data array. If you want to implement a different behavior, you can easily build a custom component with Combobox.

**Reply by MariuzM** [reply]
will look into Combobox thank you, another way to solve this is to have a prop to load options from API when form loads.
